core/storage/feedback/gae_models.py

```diff
@@ -19,11 +19,26 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+# TODO(#13594): After the domain layer is refactored to be independent of
+# the storage layer, the disable=invalid-import will
+# be removed.
+# The following import is dangerous and should not be generally
+# used. We had to use this ignore as we need to import the domain layer
+# for type-annotation.
```

Comment:
So why don't we put it in the "if MYPY" block?

Reply:
Cannot be put inside MYPY block now. As we are moving to py3 type annotations, moving the import of feedback_domain inside if MYPY: block will mean that feedback_domain is inaccessibile during runtime and will throw errors.
In py2 style this was possible because type annotations were in the form of comments but in py3, this is not possible as type annotations are not comments now.

Reply:
OK, perhaps you should explain that in the comment. (You don't need to refer to py2, just explain why it needs to be outside the mypy block in py3.)

Reply:
Added. THanks!